### PR TITLE
Add configuration for Main Telescope survey with Star Tracker. 

### PR DESCRIPTION
**Pull request by tribeiro**

### CODE REVIEW

File: python/lsst/ts/fbs/utils/maintel/basis_functions.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    -------
+    `list` of `basis_functions.BaseBasisFunction`
+    """
+
+    sun_alt_limit = -12.0
+
+    return [
+        basis_functions.NotTwilightBasisFunction(sun_alt_limit=sun_alt_limit),
+        basis_functions.HourAngleLimitBasisFunction(RA=ra, ha_limits=ha_limits),
+        basis_functions.SlewtimeBasisFunction(nside=nside, filtername="g"),
```
Comment (edennihy) [inline python/lsst/ts/fbs/utils/maintel/basis_functions.py:99]: Does it make sense to have the three slewtimebasisfunctions for the startracker surveys?

File: python/lsst/ts/fbs/utils/maintel/basis_functions.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        basis_functions.SlewtimeBasisFunction(nside=nside, filtername="r"),
+        basis_functions.SlewtimeBasisFunction(nside=nside, filtername="i"),
+        basis_functions.MoonAvoidanceBasisFunction(nside=nside),
+        basis_functions.ZenithShadowMaskBasisFunction(
```
Comment (edennihy) [inline python/lsst/ts/fbs/utils/maintel/basis_functions.py:103]: The min_alt and max_alt limits look borrowed from the auxtel survey, just commenting here to be sure these limits apply to the maintel

**Review by edennihy — APPROVED** [review]
These changes look good, two comments for your consideration but otherwise approved.

File: python/lsst/ts/fbs/utils/maintel/basis_functions.py (hunk 2 above)
Comment (tribeiro) [reply python/lsst/ts/fbs/utils/maintel/basis_functions.py:103]: You are right that the Main Telescope can go lower than this.. But I don't want to risk it at this point.

**Pull request merged**